src/transports/websocket.ts

```diff
@@ -332,3 +332,18 @@ export class WebSocketTransport<
     }
   }
 }
+
+export class WebsocketReverseMappingTransport<
```

Comment:
Could we add tsdocs for this to explain when this should be used?

Reply:
Should be covered now!